### ISSUE TITLE
bug: Missing tabindex on video player start button [accessibility]

### ISSUE THREAD

**Issue by nfreear** [microsoft/BotFramework-WebChat#3656]
Hi,

 * Relates to :~ #3655.

There is a missing `tabindex​` attribute on video players within adaptive cards, which I can see is fixed in ['AdaptiveCards' 2020.06](https://github.com/microsoft/AdaptiveCards/releases/tag/2020.06) (2.5.0); Pull request: https://github.com/microsoft/AdaptiveCards/pull/4182.

Unfortunately, I can see that WebChat 4.11.0 still [uses the older, unfixed AdaptiveCards 1.2.6](https://github.com/microsoft/BotFramework-WebChat/blob/v4.11.0/packages/bundle/package.json#L33).

When is WebChat likely to be updated to use the newer AdaptiveCards module please?

Note, this is an accessibility issue for keyboard and non-visual users and is a blocker for them.

Thanks,

Nick

## Version

 * WebChat.js version :~ 4.10.1 / 4.11.0;
 * JS :~ https://cdn.botframework.com/botframework-webchat/4.10.1/webchat.js;

## Steps to reproduce

1. Click on '....'
1. Scroll down to '....'
1. See error

## Expected behavior

 * The user should be able to navigate to a video embedded in a Chat-bot using the `TAB` key on their keyboard, and activate the triangular _Start_ button using `Enter` or `Space` key.

## Additional context

[Bug]

**Comment by gabog** [comment]
Hi @compulim, can you please take a look at this one?

Thanks

**Comment by compulim** [comment]
We bumped to Adaptive Cards 2.5.0 in our dev build and it will be released on 4.12.0.

**Comment by nfreear** [comment]
Hi @compulim,

Good to hear. Thanks for the response,

Nick
